packages/stateless/pages/CreateProposal.tsx

```diff
@@ -29,15 +29,18 @@ export const CreateProposal = ({
 
   const proposalModuleItems = useMemo(
     () =>
-      (
-        daoInfo.proposalModules
-          .map((proposalModule) => matchAdapter(proposalModule.contractName))
-          .filter(Boolean) as ProposalModuleAdapter[]
-      ).map((adapter) => ({
-        label: t(`proposalModuleLabel.${adapter.id}`),
-        value: proposalModule,
-      })),
-    [daoInfo.proposalModules, proposalModule, t]
+      daoInfo.proposalModules
```

Comment:
This is great, but can we get a comment on here? Something like:

> // Chooses the correct proposal module (i.e., the one that matches the contract name)

Or something like that.